dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -2249,6 +2416,1221 @@ async fn cmd_db_network_list_vnics(
     Ok(())
 }
 
+// Copy some types from Steno, because steno uses pub(crate) everywhere. We
+// don't want to change Steno to make the internals public, but these types
+// should be fairly stable.
```

Comment:
We don't have to do this here but I think exporting the types from Steno (maybe under a "debug" feature to discourage general use? I don't know) would be preferable to duplicating them.  I don't think there's reason to assume these are particularly stable.  We don't change steno that much but we did change these types significantly last year and this is definitely in the bucket of things I would assume we could change in Steno without breaking people.  By duplicating them we won't realize we've broken them.

(edit: more below)